examples/hello_world/test_driver/smoke_web_engine_test.dart

```diff
@@ -32,17 +32,18 @@ void main() {
       expect(await driver.getText(titleFinder), 'Hello, world!');
     });
 
+    // TODO(htoor3): Remove skip once https://github.com/flutter/engine/pull/40904 merged.
     test('enable accessibility', () async {
       await driver.setSemantics(true);
 
       // TODO(ianh): this delay violates our style guide. We should instead wait for a triggering event.
       await Future<void>.delayed(const Duration(seconds: 2));
 
       final WebElement? fltSemantics = await driver.webDriver.execute(
-        'return document.querySelector("flt-glass-pane")?.shadowRoot.querySelector("flt-semantics")',
+        'return document.querySelector("flt-semantics")',
```

Comment:
With the DOM structure change, we can query the `flt-semantics` from the `document` directly, instead of having to grab the `shadowRoot` first.